geoips/compare_outputs.py

```diff
@@ -479,20 +496,27 @@ def text_match(output_product, compare_product):
     bool
         Return True if products match, False if they differ
     """
-    retval = subprocess.call(["diff", output_product, compare_product])
-    if retval == 0:
+    ret = subprocess.run(
```

Comment:
Updated to use `subprocess.run()` rather than `subprocess.call()`. From the documentation for `subprocess.call()`:

> Do not use stdout=PIPE or stderr=PIPE with this function. The child process will block if it generates enough output to a pipe to fill up the OS pipe buffer as the pipes are not being read from.

I also piped stderr to stdout and added this output to `LOG.debug()`.